### ISSUE TITLE
GridSearch optimizer

### ISSUE THREAD

**Issue by cryptocoinserver** [SimonBlanke/Hyperactive#43]
It would be great to have GridSearch as optimizer too. 
Would make the optimizer portfolio of hyperactive complete.

Are there plans to add it in the future? Maybe existing ones could be wrapped: https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.GridSearchCV.html

Thank you for this great project.

**Comment by SimonBlanke** [comment]
Hello @cryptocoinserver,

Nice to hear, that you like this project!
I agree, that grid-search should be part of Hyperactive in the future. All optimization algorithms of Hyperactive come from [GFO](https://github.com/SimonBlanke/Gradient-Free-Optimizers). I already opened an [issue](SimonBlanke/Gradient-Free-Optimizers/issues/10) several months ago that is currently assigned to @djokester. I'm not sure what progress he has made so far. 

I will not wrap the scikit-learn algorithm, because Hyperactive works very different and I already have some ideas to make the grid-search more fun to use.

I will close this issue, because it is a duplicate of the GFO-issue. If you have more questions or suggestions you can post them there: [GFO issue](SimonBlanke/Gradient-Free-Optimizers/issues/10)
